spec/features/authorization_of_conversation.rb

```diff
@@ -0,0 +1,42 @@
+require 'rails_helper'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.